runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -1017,6 +1041,129 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 #endif /* J9VM_ENV_DATA64 */
 	}
 
+	/**
+	 * Get preserved array layout for the given indexable object
+	 *
+	 * @param forwardedHeader Forwarded header of dstObjectPtr
+	 * @param dstObjectPtr    Pointer to the indexable object whose size is required
+	 */
+	MMINLINE ArrayLayout
+	getPreservedLayout(MM_ForwardedHeader *forwardedHeader, J9IndexableObject *dstObjectPtr)
```

Comment:
I believe, we can remove this now and use getPreservedArrayLayout, instead

Reply:
That's true, missed that